experimental/cmd/ccs811/main.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	"periph.io/x/periph/conn/i2c/i2creg"
+	"periph.io/x/periph/experimental/devices/ccs811"
+	"periph.io/x/periph/host"
+)
+
+func main() {
+
+	// Make sure periph is initialized.
+	if _, err := host.Init(); err != nil {
+		log.Fatal(err)
+	}
+
+	// Use i2creg I²C bus registry to find the first available I²C bus.
+	b, err := i2creg.Open("")
```

Comment:
Please use the flag package, and add flags for the I²C bus selection and verbosity. Don't always open the default i2c bus. See other tools for examples.